store_config_in_the_environment.md

```diff
@@ -56,6 +56,21 @@ The environment is not part of the application. That's why it serves our purpose
 
 In the example below, a special tool [`EnvironmentVarGuard`](https://docs.python.org/3/library/test.html#test.support.EnvironmentVarGuard) (part of Python's standard library) is used to simulate changes in the environment from within the test cases.
 
```

Comment:
I added a code example here and in the paragraph below, it helped me understand the text better to have the code there